src/main/java/com/google/cloud/teleport/templates/common/SplunkConverters.java

```diff
@@ -111,6 +111,11 @@ public static FailsafeStringToSplunkEvent failsafeStringToSplunkEvent(
     ValueProvider<Integer> getParallelism();
 
     void setParallelism(ValueProvider<Integer> parallelism);
+  
+    @Description("Determines whether the template forwards a PubSubMessage or just the underlying data.")
+    ValueProvider<Boolean> getIncludePubSubMessage();
```

Comment:
Can we correct the capitalization here?

getIncludePubsubMessage
setIncludePubsubMessage

i.e. We want to be consistent in referring to Pubsub without the upper case S.